components/observations/ObservationDetailView.tsx

```diff
@@ -93,13 +95,44 @@ export const TableRow = ({label, value}: {label: string; value: string}) => (
   </HStack>
 );
 
+export const WeatherCard = ({observation, ...props}: {observation: Observation} & CardProps) => {
+  const {weather = {}, weather_summary = ''} = observation.advanced_fields || {};
+  // observation.advanced_fields.weather might be missing, or might be an object that's filled with empty strings -
+  // in either of those cases, we don't have weather data to render.
+  const hasWeatherEntries = Object.entries(weather || {}).some(([_k, v]) => Boolean(v));
+
+  if (!hasWeatherEntries && !weather_summary) {
+    return null;
+  }
+
+  const {cloud_cover, air_temp, recent_snowfall, rain_elevation, snow_avail_for_transport, wind_loading} = weather;
+
+  return (
+    <Card borderRadius={0} borderColor="white" header={<BodyBlack>Weather</BodyBlack>} {...props}>
+      <VStack space={8} width="100%">
+        {/* Using Boolean() here so that we don't end up rendering empty strings inline */}
+        {Boolean(weather_summary) && <HTML source={{html: weather_summary}} />}
```

Comment:
TIL `Boolean()` for when you want _extra_ truthiness!

Reply:
it's all about what you want the result to be when `weather_summary` is an empty string...

```
❯ node
Welcome to Node.js v19.4.0.
Type ".help" for more information.
> '' && "something"
''
> Boolean('') && "something"
false
```

Reply:
do you want `false`, or just falsy? 😵